core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -76,20 +88,27 @@ private static HttpEncodingType determineEncoding(String acceptEncoding) {
             }
             if (encoding.contains("*")) {
                 starQ = q;
+            } else if (encoding.contains("br") && q > brQ) {
+                brQ = q;
```

Comment:
How about assigning this only when `Brotli.isAvailable()` returns `true`?
If so, I think we can simplify the if-else branch below at line 99.

Reply:
I changed a few things. Can you review it once again? 🙇‍♂️